### PR TITLE
[Fortran] enable gfortran masklr_2

### PR DESCRIPTION
**Pull request by jeanPerier**
Fixed by https://github.com/llvm/llvm-project/pull/87496

### CODE REVIEW

**Review by tarunprabhu — APPROVED** [review]
Thanks. Works on Linux on x86. 

I have requested a review from Tom so he can make sure it works on aarch64. Please wait for his approval before merging.

**Review by tblah — APPROVED** [review]
Works for me on aarch64. Thanks!

**Comment by jeanPerier** [comment]
Thanks for testing!

**Pull request merged**
